step-capstone/src/components/Suggestions/SuggestionBar.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Grid, IconButton } from '@material-ui/core'
+import { ArrowLeft, ArrowRight } from '@material-ui/icons'
+import "../../styles/Suggestions.css"
+import SuggestionItem from "./SuggestionItem"
+
+export default class SuggestionBar extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      suggestions: props.suggestions,
+      startIndex: 0
+    }
+
+    this.loadPrevious = this.loadPrevious.bind(this);
+    this.loadNext = this.loadNext.bind(this);
+  }
+
+  loadPrevious() {
+    let newStartIndex = this.state.startIndex - 1;
+    this.setState({ startIndex: newStartIndex })
+  }
+
+  loadNext() {
+    let newStartIndex = this.state.startIndex + 1;
+    this.setState({ startIndex: newStartIndex })
+  }
+
+
+
+  render() {
+    return (
+      <div id="suggestion-bar">
+        <Grid id="suggestion-grid" container direction="row" wrap="nowrap" justify="center" >
+          <Grid item>
+            <IconButton
+              className="arrow-buttons" 
+              onClick={this.loadPrevious}
+              disabled={(this.state.startIndex === 0) ? true : false}
+            >
+              <ArrowLeft />
+            </IconButton>
+          </Grid>
+
+          <Grid item container id="displayed-suggestions" direction="row" wrap="nowrap" justify="space-around">
+            <Grid item className="suggestion-item-holder">
+              <SuggestionItem 
+                suggestion={this.state.suggestions[this.state.startIndex]} 
+                context={this.props.context}
+                onAddItem={this.props.onAddItem}
+              />
+            </Grid>
+            <Grid item className="suggestion-item-holder">
+              <SuggestionItem 
+                suggestion={this.state.suggestions[this.state.startIndex + 1]}
+                context={this.props.context}
+                onAddItem={this.props.onAddItem}
+              />
+            </Grid>
+            <Grid item className="suggestion-item-holder">
+              <SuggestionItem 
+                suggestion={this.state.suggestions[this.state.startIndex + 2]} 
+                context={this.props.context}
+                onAddItem={this.props.onAddItem}
+              />
+            </Grid>
+          </Grid>
```

Comment:
This seems a little awkward. What if you wanted to display 2 items instead, or 5? You'd have to copy-paste these blocks each time you make a change. If you can, a generator loop would be cleaner here.

Reply:
We could have a page index, which, multiplied by the number of places we want to display at a time would give us the position of the first result on that page and loop from that index however many places we want to display at a time. That way in the case we do want to change it all we have to change is the number we want to display.

Reply:
sorry that was super wordy haha

Reply:
good idea all, it is now added, thanks!